agent/engine/execcmd/manager_start_linux.go

```diff
@@ -135,35 +140,49 @@ func (m *manager) doStartAgent(ctx context.Context, client dockerapi.DockerClien
 	}
 	execRes, err := client.CreateContainerExec(ctx, containerId, execCfg, dockerclient.ContainerExecCreateTimeout)
 	if err != nil {
+		execMD.StoppedReason = fmt.Sprintf("create start error: %v", err)
```

Comment:
can we add more customer-friendly stopped reason since this is directly used by customers? same comment for other places too.